nomad/__init__.py

```diff
@@ -4,15 +4,17 @@
 
 class Nomad(object):
 
-    def __init__(self, host='127.0.0.1', port=4646, timeout=5, region=None, version='v1', verify=False, cert=()):
-        self.host = host
+    def __init__(self, uri='http://127.0.0.1', port=4646, nspace='', token='', timeout=5, region=None, version='v1', verify=False, cert=()):
```

Comment:
to avoid a breaking change with named arguments, we should still leave host and add logic to determine if its http or https and add it to the prefix, but still just pass it to the Requester as uri

Reply:
I will make a change to propose and review with you